### PR TITLE
 Add configuration for recording the radiant timing. Make it independent from the bias scan

### PR DESCRIPTION
**Pull request by fschlueter**
And PR the third! My intention is in the title. Unfortunately there are a lot of whitespace changes in this PR. But you can remove them from the diff on GitHub. 

I just copied the code for bias_scan, will that work?

### CODE REVIEW

**Comment by fschlueter** [comment]
You can also just inspect the last commit

**Comment by fschlueter** [comment]
We could try that at station 12.

**Review by cozzyd — APPROVED** [review]
looks good to me!

**Comment by fschlueter** [comment]
I should add entries to the configs I guess?
I know the default values will be used ...

File: src/rno-g-acq.c [hunk 1]
```diff
@@ -1637,6 +1641,10 @@ static void * wri_thread(void* v)
 
   }
 
+  if (runinfo)
+  {
+    fprintf(runinfo, "TOTAL-NUMBER-OF-EVENTS-WRITTEN = %d\n", num_events);
+  }
```
Comment (fschlueter) [inline src/rno-g-acq.c:1647]: grrr what is that doing here.....

**Comment by fschlueter** [comment]
This is running of 14 now. If everything looks good tmr I will enrol it to all other stations. 

I do not have to stop the daq for this right?

**Comment by cozzyd** [comment]
no need to stop DAQ, if you `make install` will run on next invocation

**Pull request merged**